op-node/rollup/interop/managed/api.go

```diff
@@ -0,0 +1,56 @@
+package managed
+
+import (
+	"context"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	gethrpc "github.com/ethereum/go-ethereum/rpc"
+
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+	supervisortypes "github.com/ethereum-optimism/optimism/op-supervisor/supervisor/types"
+)
+
```

Comment:
Is this API in the spec?

Reply:
Managed Nodes (and their API) are not yet in the spec. I would consider Managed mode to be more of an implementation detail of the Supervisor than a spec behavior of the node. If we wanted other clients to support a Managed Mode, we'll need to describe the total set of behaviors that need to change, which is still evolving as we land it.